src/client/client.cpp

```diff
@@ -514,6 +514,7 @@ void Client::step(float dtime)
 		std::vector<v3s16> blocks_to_ack;
 		while (!m_mesh_update_thread.m_queue_out.empty())
 		{
+			m_new_mesh_count++;
```

Comment:
Alternatively you could `m_new_mesh_count += num_processed_meshes` at the end of the parent scope.
doesn't really matter though